crypto/dkg_test.go

```diff
@@ -181,13 +189,17 @@ func dkgCommonTest(t *testing.T, dkg int, n int, threshold int, test testCase) {
 	mrand.Seed(rand)
 	t.Logf("math rand seed is %d", rand)
 	var r1, r2 int
+	var h int // h is the offset of the honest nodes to check the final protocol results are correct
```

Comment:
If `h` is 2 and there are 4 nodes (1,2,3,4), then node 3 and 4 are honest?

Reply:
Yes that's correct, I'll update the comment.